jetbrains-core/src/software/aws/toolkits/jetbrains/utils/PsiUtils.kt

```diff
@@ -10,11 +10,16 @@ import com.intellij.psi.PsiElement
 fun PsiElement.isTestOrInjectedText(): Boolean {
     val project = this.project
     val virtualFile = this.containingFile.virtualFile ?: return false
-    if (virtualFile is VirtualFileWindow) {
+    if (this.isInjectedText() || ProjectRootManager.getInstance(project).fileIndex.isInTestSourceContent(virtualFile)) {
```

Comment:
`ProjectRootManager.getInstance(project).fileIndex.isInTestSourceContent(virtualFile)`
what does this mean?

Reply:
I'm not sure what's the use case for this, I didn't touch this part from the original function, I just extracted the injected check into a separate function so we can use

Reply:
ah I see below